### ISSUE TITLE
Video Speed Controller (VSC) icon appearing in center of video instead of corner

### ISSUE THREAD

**Issue by Gitoffthelawn** [codebicycle/videospeed#166]
For some videos, VSC's icon appears right smack in the middle of the video!

Here's a good example:  
https://old.reddit.com/r/aww/comments/rc2w4g/disabled_mother_trained_her_dog_to_pick_up_her/

VSC 0.6.3.3 in Firefox 95.0.0

**Comment by ACryingMan** [comment]
I think it only occur in firefox 95, does not appear on my edge and safari.

**Comment by magnus-ISU** [comment]
Additionally, cannot drag the controller. It happens anywhere on old reddit it seems. Does it happen on other websites?

**Comment by magnus-ISU** [comment]
In `inject.js` line 276, we set `top` and `left` variables, I think debugging there will reveal more about why this happens. Also, unrelated to this bug but at line 350 `www.reddit.com` should match all reddits

**Comment by magnus-ISU** [comment]
As far as I can tell, hardcoding those values to `0px` instead of performing any calculation results in the correct behavior. It doesn't break anything on odysee, youtube, facebook, Crunchyroll. Twitter didn't work before or after (EDIT: works after, forgot to disable the blocklist when I tested). So I have absolutely no idea why it was like that before. @codebicycle you were the last person to edit that, what was the reasoning/need for it?

**Comment by ACryingMan** [comment]
![Screenshot 2021-12-17 at 20 18 30](https://user-images.githubusercontent.com/26747671/146543729-b566bdf0-9e44-4edc-888f-a6326116aa65.png)

Hi sir, as you can see it still appear in the middle of the screen in netflix on firefox 95. I have uninstall and reinstall the extension.

**Comment by Gitoffthelawn** [comment]
Here is another site that can be used as an example of this issue: https://www.epicgames.com/store/p/antstream-arcade

**Comment by magnus-ISU** [comment]
@ACryingMan my apologies for not being clear. I finished work on the extension yesterday, but did not publish it until now for various reasons. In about 24 hours (once mozilla approves it), you should be able to go to https://addons.mozilla.org/en-US/firefox/addon/videospeed-magnus-fork/ to install my version of the addon, which has this fixed as you can see for example with @Gitoffthelawn's site:

![image](https://user-images.githubusercontent.com/75949275/146602130-9f529654-9a41-4c6f-990d-2349ad879337.png)

It has some other changes as well. I have submitted these to the upstream chromium extension, but that hasn't been changed in 2 months. This firefox version hasn't been updated for 8 months. So I expect it will take a long time for the current, popular version of the extension to see the improvements I have created.

Like I said though, you should be able to install from the link I gave there and have it fixed tomorrow.

**Comment by Gitoffthelawn** [comment]
@magnus-ISU  Thanks for publishing on AMO.  Not available yet, but likely in the next 12-15 hours.  I didn't realize they were requiring approval of addons again.  Probably a good thing, because they were getting a whole bunch of spam addons.

For those interested, here is a link to the GitHub fork created by Magnus:  https://github.com/magnus-ISU/videospeed

**Comment by magnus-ISU** [comment]
@Gitoffthelawn @ACryingMan It is listed, so you can go download it. Please if it doesn't work as expected on any sites open an issue on my repo, I will see if I can deal with it. The way I fixed this issue is different from the way the chromium upstream did it.

**Comment by Gitoffthelawn** [comment]
@magnus-ISU Thank you.  I'm hoping to take a look at it when I have a little time.

**Comment by Gitoffthelawn** [comment]
@magnus-ISU It appears you have the issue reporter disabled in your repo.

**Comment by magnus-ISU** [comment]
@Gitoffthelawn Thanks, sorry. Fixed.

**Comment by ACryingMan** [comment]
@magnus-ISU Hi sir, thank you so much for the update. Its been working perfectly as intended. Appreciate it

**Comment by Blagor1990** [comment]
I believe the issue appears again.

However, it only takes place when I open a Netflix video by using Firefox 96.0.3. With Google Chrome, it works fine.

**Comment by deadpahn** [comment]
yeah, It's in the middle for me too on netflix. Can't drag it off the screen. 91.8.0esr (64-bit)
